.claude/rules/code-style.md

```
@@ -0,0 +1,422 @@
+- `@/lib/db/*` — Database access
+- `@clerk/nextjs/server` — Server-side auth
+- `stripe`, `resend` — API clients with secrets
+- `drizzle-orm` — ORM queries
+- `*.server.ts` files
+
+**Commands:**
+- `pnpm --filter web lint:server-boundaries` — Quick check for boundary violations
+- `pnpm --filter web lint:eslint` — Full ESLint check
+
+**Error Messages:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)